package.json

```diff
@@ -60,7 +60,7 @@
 		"pretest": "npm run -s install-if-no-packages",
 		"test": "./node_modules/jest-24.9.0/bin/jest.js --config tests/js/jest.config.json",
 		"test:debug": "node --inspect-brk ./node_modules/jest-24.9.0/bin/jest.js --config tests/js/jest.config.json --watch --runInBand --no-cache",
-		"test:e2e": "WP_VERSION=latest npm run build && npx wc-e2e docker:down && npx wc-e2e docker:up && npx wc-e2e test:e2e",
+		"test:e2e": "npm run build && [[$(docker ps | grep woocommerce-admin) == '']] || npx wc-e2e docker:down && npx wc-e2e docker:up && npx wc-e2e test:e2e",
```

Comment:
Could a more unique string than `woocommerce-admin` be used? I think this will also match other Docker services we run.

Reply:
> Could a more unique string than woocommerce-admin be used?

Yes, we can change it with an `appName` entry in `tests/e2e/config/default.json`. Would `woocommerce-admin-e2e` work?

Reply:
@adrianduffell I updated the PR to change the container prefix to `woocommerce-admin-e2e`. The small catch that goes with the change is that people will need to stop their existing e2e containers with `docker-compose`.

Reply:
@rrennick It doesn't seem ideal to use `docker-compose` to stop the containers, but I don't think it is a blocker for this PR.

What's the reason for it? Could it be improved in the future?

Reply:
> What's the reason for it?

It's a one time thing. The current containers (if someone has them running) are prefixed with `woocommerce-admin`. The script to stop them will be looking for `woocommerce-admin-e2e`. Since the old prefix doesn't match, the e2e `docker:down` won't attempt to stop them. The new ones won't be able to start because the port is already allocated.

Reply:
Ah I see, thanks,I didn't understand it would be a one time thing. That sounds perfectly fine!